lib/pure/os.nim

```diff
@@ -1752,17 +1748,18 @@ proc isAdmin*: bool {.noWeirdTarget.} =
                                               addr administratorsGroup)):
       raiseOSError(osLastError(), "could not get SID for Administrators group")
 
-    defer:
```

Comment:
Out of curiosity, why move away from `defer`?

Reply:
I don't like, it's the only construct in Nim that lacks visible scoping rules.